opentelemetry-otlp/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Instead of using
 `.with_config(Config::default().with_resource(RESOURCE::default()))` users must
 now use `.with_resource(RESOURCE::default())` to configure Resource when using
 `OtlpLogPipeline`.
+- The methods `OtlpTracePipeline::install_simple()` and `OtlpTracePipeline::install_batch()` now return `TracerProvider` instead of `Tracer`.
```

Comment:
also callout global is not being set, and it is user responsibility to do so.
its best to show before and after code the user must do to ensure same behavior as before.

Reply:
Also good to mention this as breaking change.

Reply:
Updated